ResearchKit/Consent/ORKSignatureView.m

```diff
@@ -254,6 +259,23 @@ - (NSArray *)backgroundLines {
 
 #pragma mark Touch Event Handlers
 
+- (BOOL)_isForceTouchAvailable {
+    static BOOL isAvailable;
+    static dispatch_once_t onceToken;
+    
+    dispatch_once(&onceToken, ^{
+        
+        isAvailable = NO;
+        if ([self.traitCollection respondsToSelector:@selector(forceTouchCapability)]) {
+            if (self.traitCollection.forceTouchCapability == UIForceTouchCapabilityAvailable) {
```

Comment:
You can fold both checks onto the same `if` line as:

    if ([self.traitCollection respondsToSelector:@selector(forceTouchCapability)] && self.traitCollection.forceTouchCapability == UIForceTouchCapabilityAvailable)`.

The second test won't be run unless the first one passes.